ReactSkia/views/common/RSkImageCacheManager.cpp

```diff
@@ -20,121 +21,91 @@ using namespace std;
 
 namespace facebook {
 namespace react {
+namespace ImageCacheManager {
 
-namespace RSkImageCacheManager{
-   
-namespace {
+#define CPU_MEM_ARR_INDEX 0
+#define GPU_MEM_ARR_INDEX 1
 
-  typedef better::map <string, sk_sp<SkImage>> ImageCacheMap;
-  ImageCacheMap imageCache;
-  #define CPU_MEM_ARR_INDEX 0
-  #define GPU_MEM_ARR_INDEX 1
-  std::mutex mutexLock;
+RSkImageCacheManager::RSkImageCacheManager() {};
```

Comment:
if there is no definition for constructor and destructor , this two lines are not required.As it is behaving like default constructor/destructor itself.